src/Middleware/HttpOverrides/src/CertificateForwarderExtensions.cs

```diff
@@ -2,13 +2,14 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-
 using Microsoft.AspNetCore.HttpOverrides;
 using Microsoft.Extensions.DependencyInjection;
-using System.Security.Cryptography.X509Certificates;
 
 namespace Microsoft.AspNetCore.Builder
 {
+    /// <summary>
+    /// Extension methods for using the certificate fowarder
```

Comment:
Need full proper grammar (i.e. complete sentences), so need to end with a period.